LICENSE-binary

```diff
@@ -279,6 +302,9 @@ io.netty:netty-transport-native-epoll
 io.netty:netty-transport-native-kqueue
 io.netty:netty-transport-native-unix-common
```

Comment:
I remember the kqueue and native-unix removed from tgz, could you please check that?

Reply:
> I remember the kqueue and native-unix removed from tgz, could you please check that?

Sure, had checked. `io.netty:netty-transport-native-kqueue` removed and `native-unix` still contains.
I will remove `kqueue`.